string detection_type
observation[] observations